mypy/stubtest.py

```diff
@@ -1009,6 +968,78 @@ def verify_typealias(
     )
 
 
+# ====================
+# Helpers
+# ====================
+
+
+IGNORED_MODULE_DUNDERS = frozenset(
+    {
+        "__file__",
+        "__doc__",
+        "__name__",
+        "__builtins__",
+        "__package__",
+        "__cached__",
+        "__loader__",
+        "__spec__",
+        "__path__",  # mypy adds __path__ to packages, but C packages don't have it
+        "__getattr__",  # resulting behaviour might be typed explicitly
+        # TODO: remove the following from this list
+        "__author__",
+        "__version__",
+        "__copyright__",
```

Comment:
```suggestion
        "__copyright__",
        "__about__",
```

Another one I noticed in my experiments recently!

Reply:
Oh, I see you consider `heap.__about__` a true positive?

Reply:
I guess we could just put `__about__: str` in the stub. Doesn't do any harm.

Reply:
It's quite educational :D

Reply:
My reasoning here is that it's only clearly a true negative if the Python runtime is what's creating the attribute. Everything else you might conceivably want to have stubbed. I'm especially eager to remove `__version__` from this list, which is widely used in third party stubs.

Reply:
Sure, that makes sense!